source/winUser.py

```diff
@@ -467,9 +467,11 @@ def setForegroundWindow(hwnd):
 def setFocus(hwnd):
 	user32.SetFocus(hwnd)
 
-def getDesktopWindow():
+
+def getDesktopWindow() -> HWND:
 	return user32.GetDesktopWindow()

```

Comment:
Does this return `HWND` (A ctypes pointer) or is it converted to an int, I.E. what we've called HWNDVal elsewhere?

Reply:
fixed this, it should be `HWNDVal`